keyform/models.py

```diff
@@ -47,7 +52,10 @@ class Request(models.Model):
     bpn = models.CharField(max_length=9, verbose_name=_('M-Number'), validators=[bpn_validator])
     created_timestamp = models.DateTimeField(default=now, blank=True)
     charged_on_rcr = models.BooleanField(default=False, verbose_name=_('Charged on RCR'))
-    status = models.CharField(max_length=2, choices=STATUS_TYPES, default = 'pr')
+    status = models.CharField(max_length=2, choices=STATUS_TYPES, default='pr')
+    previous_status = models.CharField(max_length=2, choices=STATUS_TYPES)
```

Comment:
Should this be nullable? Or when it is created do we set the status and previous status to the same thing?